[directxtk, directxtk12, directxtex] ports updated for August 2021 release

walbourn opened the pull request:
Updated the **directxtk**, **directxtk12**, and **directxtex** ports to the latest GitHub releases.

> Changed use of FWLINKs back to direct GitHub links per feedback from my last PR.

walbourn committed:
DirectXTex port updated

walbourn committed:
DirectX Tool Kit updated for August 2021 release

walbourn committed:
Update baseline

JonLiu1993 commented:
@walbourn ,Could you please take a look:
[build-x86-windows-rel-out.log](https://github.com/microsoft/vcpkg/files/6923365/build-x86-windows-rel-out.log)

walbourn commented:
I've updated MakeSpriteFont to use .NET 4.7.2 instead of .NET 4.6. 

Can your build machine be updated to have the 4.7.2 targeting pack installed?

I updated it because the 4.6 version won't build with a system with only VS 2019 installed. 4.6 dates back to VS 2015.

walbourn committed:
Remove build of MakeSpriteFont

walbourn committed:
Update baseline

JonLiu1993 commented:
> I've updated MakeSpriteFont to use .NET 4.7.2 instead of .NET 4.6.
> 
> Can your build machine be updated to have the 4.7.2 targeting pack installed?
> 
> I updated it because the 4.6 version won't build with a system with only VS 2019 installed. 4.6 dates back to VS 2015.
> 
> > A workaround is that I update the _directxtk_ port to always download MakeSpriteFont instead of trying to build it which is what I do in **directxtk12**...

@BillyONeal ,Could you please take a look?

BillyONeal commented:
>Can your build machine be updated to have the 4.7.2 targeting pack installed?

The script we use to mint the build machine images is https://github.com/microsoft/vcpkg/blob/master/scripts/azure-pipelines/windows/create-vmss.ps1 -- you specifically would probably want to edit https://github.com/microsoft/vcpkg/blob/master/scripts/azure-pipelines/windows/deploy-visual-studio.ps1 ?

walbourn committed:
Added 4.7.2 .NET TargetingPack

walbourn commented:
@JonLiu1993 Is there something blocking this PR?

JonLiu1993 commented:
> @JonLiu1993 Is there something blocking this PR?

No,will merge soon

walbourn commented:
Ping?

BillyONeal committed:
Back out adding .NET targeting pack, forked out as https://github.com/microsoft/vcpkg/pull/19764
